examples/darwin-framework-tool/commands/delay/WaitForCommissioneeCommand.mm

```diff
@@ -22,10 +22,7 @@
 
 CHIP_ERROR WaitForCommissioneeCommand::RunCommand()
 {
-    MTRDeviceController * commissioner = CurrentCommissioner();
-    VerifyOrReturnError(nil != commissioner, CHIP_ERROR_INCORRECT_STATE);
-
-    auto * base_device = [MTRBaseDevice deviceWithNodeID:@(mNodeId) controller:commissioner];
+    auto * base_device = BaseDeviceWithNodeId(mNodeId);
```

Comment:
Probably does not hurt, but not sure WaitForCommissioneeCommand makes sense over PASE....